doc_src/string.txt

```diff
@@ -48,6 +49,8 @@ The following subcommands are available:
 
 - `replace` is similar to `match` but replaces non-overlapping matching substrings with a replacement string and prints the result. By default, PATTERN is treated as a literal substring to be matched. If `-r` or `--regex` is given, PATTERN is interpreted as a Perl-compatible regular expression, and REPLACEMENT can contain C-style escape sequences like `\t` as well as references to capturing groups by number or name as `$n` or `${n}`. Exit status: 0 if at least one replacement was performed, or 1 otherwise.
 
+- `repeat` repeats the STRING `-t` or `--times` times. If `-n` or `--no-newline` is given, the output won't contain a newline character at the end. Exit status: 0 if yielded string is not empty, 1 otherwise.
```

Comment:
I think this should be `-c` and `--count` for consistency with other fish commands. You should also document what happens if the repeat value is omitted or explicitly set to zero.